front/lib/registry.ts

```diff
@@ -5,6 +5,7 @@ import config from "@app/lib/api/config";
 
 export const PRODUCTION_DUST_WORKSPACE_ID = "0ec9852c2f";
 export const PRODUCTION_DUST_APPS_WORKSPACE_ID = "78bda07b39";
+export const PRODUCTION_DUST_APPS_VAULT_ID = "vlt_rICtlrSEpWqX";
```

Comment:
we'll have to update `createActionRegistry` to update the value of appVaultId  with a local value for development, along with workspaceId.

Reply:
fixed 👍